### ISSUE TITLE
Type Checker cannot determine type for pattern of 1 in cases statement

### ISSUE THREAD

**Issue by joey-coleman** [symphonytool/symphony#90]
> The following bug was [originally reported](https://sourceforge.net/p/compassresearch/bugs/90/) on Sourceforge by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-10-03 13:25:25.570000:

channels 
a, 
b : int

process A = 
begin 
    state
        v : int := 2

```
operations

    Test : () ==> ()
    Test() == 
                                cases v:
"Could not determine type for 1."--->   1 -> v:=1,
                                0 -> v:=0,
                                others -> v:=6
                                end

@ a -> v := v + 1;Test(); b.v -> Skip
```

end

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/90/#79db) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-10-08 11:30:21.837000:
- **status**: open --> closed

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/90/#18a2) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-10-08 11:30:22.196000:

fixed in [70859186de4338a24ce5ef42dbf6e8efc175f258]
